WORKSPACE

```diff
@@ -37,10 +37,10 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
-# envoy-wasm commit date: Feb 28 2020
-ENVOY_SHA = "f4ace91833442010118a592c2dff3689d984337d"
+# envoy-wasm commit date: Mar 3 2020
+ENVOY_SHA = "373eafdb20958a019f5556e508b5286eba88af06"
```

Comment:
Could you grab `e64efd960cd8e97287764b9bae39b3a0def95046` while you're at it?

Reply:
It might be good to limit 1.5 changes for cherry-picking.

Reply:
do workspace update. in a separate pr?

Reply:
1) This is `master` branch, so it doesn't matter in terms of cherry-picking into `release-1.5`, other than for the fact that it would get tested sooner, rather than later.
2) That change is getting into `release-1.5` along with few other WASM fixes anyway.

Reply:
Why separate PR? It will just conflict with this.

Reply:
I had created https://github.com/istio/proxy/pull/2744 pr for updating sha, will skip it then and just update sha here

Reply:
Updated SHA here. only